.travis.yml

```diff
@@ -1,26 +1,38 @@
+dist: xenial
 language: c
-dist: trusty
 sudo: required
 
 env:
-  - PGUSER=postgres PGDATABASE=postgres
+  global:
+    - PGUSER=postgres
+    - PGDATABASE=postgres
+    - PGOPTIONS='-c client_min_messages=NOTICE'
+    - PGPORT=5432
+    - POSTGIS_VERSION="2.5"
+
+  matrix:
+    - POSTGRESQL_VERSION="9.5"
```

Comment:
Maybe we can drop 9.5 now?

Reply:
Maybe in the future. Since it's working right now I'll leave it for now